package.json

```
@@ -18,12 +18,11 @@
   "scripts": {
     "start:ios:dev": "react-native run-ios --configuration Debug",
     "start:ios:prod": "react-native run-ios --configuration Release",
-    "start:android:dev": "react-native run-android --variant devDebug",
-    "start:android:prod": "react-native run-android --variant prodRelease",
+    "start:android:dev": "react-native run-android --variant debug",
+    "start:android:prod": "react-native run-android --variant release",
     "build:android:js:dev": "react-native bundle --platform android --dev true --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle",
     "build:android:js:prod": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle",
     "build:android:aab:prod": "cd android && ./gradlew bundleProdRelease || cd -",
```

We should change this one too, sorry! 🙏 

```suggestion
    "build:android:aab:prod": "cd android && ./gradlew bundleRelease || cd -",
```